app/src/main/res/values-de/string.xml

```diff
@@ -0,0 +1,139 @@
+<resources xmlns:tools="http://schemas.android.com/tools">
+
+    <!-- login -->
+    <string name="login_heading1">Einloggen in deinen</string>
+    <string name="login_heading2">Blog</string>
+
+    <string name="prompt_blog_url">Blogadresse (http://blog.beispiel.de)</string>
+    <string name="prompt_email">Email</string>
+    <string name="prompt_password">Passwort</string>
+    <string name="action_sign_in">Anmelden</string>
+
+    <string name="error_invalid_url">Diese Url ist ungültig.</string>
+    <string name="error_invalid_email">Diese Emailadresse ist ungültig.</string>
+    <string name="error_invalid_password">Dieses Passwort ist zu kurz.</string>
+    <string name="error_field_required">Bitte fülle dieses Feld aus.</string>
+
+    <string name="no_such_blog">Es konnte kein Ghost Admin-Panel in %s/ghost gefunden werden.</string>
+    <string name="login_unexpected_error">Unerwarteter Fehler beim Anmelden. Bitte versuche es erneut.</string>
+    <string name="login_connection_error">Es konnte keine Verbindung zu %s/ghost erstellt werden. Bitte versuche es erneut.</string>
+    <string name="login_ssl_unsupported">Dein Blog unterstützt leider kein https:// (SSL). Versuch es stattdessen mit http://.</string>
+    <string name="logout">Abmelden</string>
+    <string name="password_changed">Hat sich dein Passwort geändert? Anmeldung mit diesem Passwort nicht möglich.</string>
+    <string name="unsynced_changes_msg">Alle nicht synchronisierten Änderungen gehen verloren, wenn du dich jetzt abmeldest</string>
+    <string name="dont_logout">Nicht abmelden</string>
+
+    <!-- posts -->
+    <string name="status_published">Veröffentlicht %s</string>
+    <string name="status_draft">Entwurf</string>
+    <string name="status_scheduled">Geplant - %s</string>
+    <string name="status_offline_changes">Auf dem Gerät gespeichert, wartet auf eine Verbindung…</string>
+    <string name="status_published_auto_saved">Änderungen automatisch gespeichert - noch nicht veröffentlicht</string>
+    <string name="status_marked_for_deletion">Zum Löschen markiert, wartet auf eine Verbindung</string>
+    <string name="status_marked_for_deletion_open_error">Dieser Artikel wurde zum Löschen markiert, du kannst ihn nicht öffnen.</string>
+    <string name="post_limit_exceeded" tools:ignore="PluralsCandidate"><![CDATA[%1$s Im Moment werden nur die letzten %2$d Artikel von deinem Blog geladen. Falls du gerne die Möglichkeit hättest auch ältere Posts zu laden, <a href="%3$s">gib mir deine Stimme bei Github</a> und hilf mir die Entwicklung dafür zu priorisieren.]]></string>
+    <string name="send_feedback">Sende Feedback…</string>
+
+    <string name="preview">Vorschau</string>
+    <string name="edit">Bearbeiten</string>
+    <string name="publish">Veröffentlichen</string>
+    <string name="update_post">Artikel aktualisieren</string>
```

Comment:
Do you refer to "Artikel aktualisieren? This could also just be
"Aktualisieren" as well. Might look better in app, you're right.

Am 6. Juli 2016 um 21:01 schrieb Vicky Chijwani <notifications@github.com>:

> In app/src/main/res/values-de/string.xml
> <https://github.com/vickychijwani/quill/pull/142#discussion_r69791533>:
>
> > +
> > +    <!-- posts -->
> > +    <string name="status_published">Veröffentlicht %s</string>
> > +    <string name="status_draft">Entwurf</string>
> > +    <string name="status_scheduled">Geplant - %s</string>
> > +    <string name="status_offline_changes">Auf dem Gerät gespeichert, wartet auf eine Verbindung…</string>
> > +    <string name="status_published_auto_saved">Änderungen automatisch gespeichert - noch nicht veröffentlicht</string>
> > +    <string name="status_marked_for_deletion">Zum Löschen markiert, wartet auf eine Verbindung</string>
> > +    <string name="status_marked_for_deletion_open_error">Dieser Artikel wurde zum Löschen markiert, du kannst ihn nicht öffnen.</string>
> > +    <string name="post_limit_exceeded" tools:ignore="PluralsCandidate"><![CDATA[%1$s Im Moment werden nur die letzten %2$d Artikel von deinem Blog geladen. Falls du gerne die Möglichkeit hättest auch ältere Posts zu laden, <a href="%3$s">gib mir deine Stimme bei Github</a> und hilf mir die Entwicklung dafür zu priorisieren.]]></string>
> > +    <string name="send_feedback">Sende Feedback…</string>
> > +
> > +    <string name="preview">Vorschau</string>
> > +    <string name="edit">Bearbeiten</string>
> > +    <string name="publish">Veröffentlichen</string>
> > +    <string name="update_post">Artikel aktualisieren</string>
>
> Should this string be made shorter? It's double the size of the English
> string so it looks a little awkward to me, on a button (screenshot below).
> But if you are generally used to long strings in apps then I'm 100% ok with
> it 😄
>
> [image: "update post" german string]
> <https://cloud.githubusercontent.com/assets/439616/16630900/ed751032-43d9-11e6-8c48-5fab588068ee.png>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/vickychijwani/quill/pull/142/files/1ed387d8935d68a67ae28264faacc5b24ec8fcb0#r69791533>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ATUI1xyOsUO46txoIKgeVrZknav77fT0ks5qS_t4gaJpZM4JGX7q>
> .
>



-- 


___________________________________________________

Dennis Mayk  |  dennis@mayk.eu
Student of Marine Geosciences at Christian-Albrechts-University, Kiel